elasticmq.conf

```diff
@@ -0,0 +1,46 @@
+// ElasticMQ README copy-pasta
```

Comment:
can we hide this file somewhere? like the `compose` folder for example. It's service-specific and it will clutter the root directory less then